packages/services/api/src/modules/shared/providers/storage.ts

```diff
@@ -95,8 +94,6 @@ export interface Storage {
   createOrganization(
     _: Pick<Organization, 'slug'> & {
       userId: string;
-      adminScopes: ReadonlyArray<OrganizationAccessScope | ProjectAccessScope | TargetAccessScope>;
-      viewerScopes: ReadonlyArray<OrganizationAccessScope | ProjectAccessScope | TargetAccessScope>;
```

Comment:
these are now defined in code, so no need to store them in the database for these roles.